hack/local-up-cluster.sh

```diff
@@ -400,6 +400,44 @@ cleanup()
   exit 0
 }
 
+# Check if all processes are still running. Prints a warning once each time
+# a process dies unexpectedly.
+healthcheck()
+{
+  if [[ -n "${APISERVER_PID-}" ]] && ! sudo kill -0 ${APISERVER_PID} 2>/dev/null; then
+    warning "API server terminated unexpectedly, see ${APISERVER_LOG}"
+    APISERVER_PID=
+  fi
+
+  if [[ -n "${CTRLMGR_PID-}" ]] && ! sudo kill -0 ${CTRLMGR_PID} 2>/dev/null; then
+    warning "kube-controller-manager terminated unexpectedly, see ${CTRLMGR_LOG}"
+    CTRLMGR_PID=
+  fi
+
+  if [[ -n "$DOCKERIZE_KUBELET" ]]; then
+    # TODO (?): check health also in this case.
+    :
```

Comment:
nit - stray `:`?

Reply:
No, the : is intentional. A statement is needed between "then" and "fi", and : is the shortest possible statement that doesn't do anything.